### DISCUSSION TITLE
Logging workflows by using Log Task

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#15086] [Q&A] [answered]
How can I log any workflows errors by using **Log Task**?

![mGQS8nJQ116cB5qu](https://github.com/OrchardCMS/OrchardCore/assets/61869142/8722fa59-2355-4c0a-a7ac-bc084b8fb02d)

Could you give me any example about this kind of task?

**Comment by weirdyang** [comment, marked as answer]
https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Workflows/Events/WorkflowFaultEvent.cs

there is a WorkflowFaultEvent activity which you can use
